tests/components/v2c/test_config_flow.py

```diff
@@ -85,3 +91,46 @@ async def test_form_cannot_connect(
     assert result3["data"] == {
         "host": "1.1.1.1",
     }
+
+
+async def test_device_id_already_exists(
+    hass: HomeAssistant,
+    mock_setup_entry: AsyncMock,
+    mock_v2c_data,
+    caplog: pytest.LogCaptureFixture,
+) -> None:
+    """Test serial number already exists."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    with patch(
+        "pytrydan.Trydan.get_data",
+        return_value=mock_v2c_data,
+    ):
+        result2 = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            {
+                "host": "1.1.1.1",
+            },
+        )
+        await hass.async_block_till_done()
+
+    assert result2["type"] is FlowResultType.CREATE_ENTRY
```

Comment:
This all can be replaced with creating a MockConfigEntry with the same unique_id and adding it to hass

Reply:
Done